src/main/java/com/swp/common/domain/Board.java

```diff
@@ -0,0 +1,32 @@
+package com.swp.common.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "boards")
+public class Board {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "board_id",nullable = false)
+    private Long boardId;
+
+    @ManyToOne
+    @JoinColumn(name ="user_id",nullable = false)
+    private Long userId;
```

Comment:
`@ManyToOne`은 객체 참조 아닌가용??

```suggestion
    @ManyToOne
    @JoinColumn(name ="user_id",nullable = false)
    private User user;
```

Reply:
아래 studyId도 마찬가지 입니다!